packages/model/addon/-private/model.js

```diff
@@ -2184,21 +2100,219 @@ Model.reopenClass({
    @param {Object} [binding] the value to which the callback's `this` should be bound
    @static
    */
-  eachTransformedAttribute(callback, binding) {
-    get(this, 'transformedAttributes').forEach((type, name) => {
+  static eachTransformedAttribute(callback, binding) {
+    this.transformedAttributes.forEach((type, name) => {
       callback.call(binding, name, type);
     });
-  },
+  }
 
   /**
    Returns the name of the model class.
 
    @method toString
    @static
    */
-  toString() {
+  static toString() {
     return `model:${get(this, 'modelName')}`;
+  }
+}
+Model.prototype._internalModel = null;
```

Comment:
Why do we do this here on the prototype?

Reply:
I think this might be to prevent a setter/getter install happening, but don't fully remember. It's been a thing here for a long time, might be able to remove now.